<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>328</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>El coche de Homer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelNombre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre del vehículo:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditNombre">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>10</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="labelRuedas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Número de ruedas:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRuedas">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelPotencia">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Potencia del motor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCombustible">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tipo de combustible:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelColor">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxColor">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>Rojo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Amarillo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Azul</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelAccesorio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accesorio:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBurbuja">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Burbuja para niños</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkVocinas">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Múltiples vocinas</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkPosavasos">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Portavasos gigante</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPotencia">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>75CV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>110CV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150CV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>170CV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200CV</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxCombustible">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>Diesel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gasolina 98</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gasolina 95</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eléctrico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Híbrido</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Solar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A pedales</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxAccesorio">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>Rueda de repuesto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kit de reparación de pinchazos</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelMatricula">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Matricula generada: </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tipo de vehículo: </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMatriculaGenerada">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTipoGenerado">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMatrcula">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Generar matrícula</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonGuardar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Crear y guardar vehiculo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
